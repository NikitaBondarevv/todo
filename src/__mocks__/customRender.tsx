import { render } from '@testing-library/react'
import { BrowserRouter } from 'react-router-dom'
import { Provider } from 'react-redux'

import { store } from 'store'

export const customRender = (Cpmponent: React.ReactNode) => render(
  <BrowserRouter>
    <Provider store={store}>
      {Cpmponent}
    </Provider>
  </BrowserRouter>
)