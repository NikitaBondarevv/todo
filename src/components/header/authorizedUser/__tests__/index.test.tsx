import { customRender } from '__mocks__/customRender'
import { AuthorizedUser } from '..'

describe('<AuthorizedUser />', () => {
  test('should match snapshot', () => {
    const { asFragment } = customRender(<AuthorizedUser />)

    expect(asFragment()).toMatchSnapshot()
  })
})
