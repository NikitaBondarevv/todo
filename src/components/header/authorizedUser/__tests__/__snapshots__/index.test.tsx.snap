// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthorizedUser /> should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="menu"
  >
    <a
      class="userName"
      href="/"
    />
    <ul
      class="dropDown"
    >
      <li>
        <a
          href="/profile"
        >
          Profile
        </a>
      </li>
      <li>
        <a
          href="/logout"
        >
          Log out
        </a>
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
